### ISSUE TITLE
`image` still depends on `jpeg-decoder` through the `tiff` crate

### ISSUE THREAD

**Issue by Shnatsel** [image-rs/image#2289]
While `image` 0.25.x has migrated to `zune-jpeg` as its JPEG implementation, it still depends on the `jpeg-decoder` crate transitively through the `tiff` crate.

To reproduce this, run `cargo tree -i jpeg-decoder` in the repository.

**Comment by gsspdev** [comment]
I get this error when trying to build on macOS:

﻿﻿﻿﻿﻿﻿﻿ Compiling jpeg-decoder v0.3.1
   Compiling rav1e v0.7.1
error: Unrecognized option: 'diagnostic-width'

error: could not compile `jpeg-decoder` (lib)
warning: build failed, waiting for other jobs to finish...

Might this be related?

**Comment by fintelia** [comment]
No, this issue is tracking a feature change. The code itself should compile as-is.

**Comment by kornelski** [comment]
@gsspdev This error happens when you have 3rd party `cargo` executable on the system (e.g. from OS package manager or Homebrew), which doesn't integrate correctly with `rustup`. Unrelated to this crate. https://github.com/rust-lang/cargo/issues/14071#issuecomment-2168024976

**Comment by Shnatsel** [comment]
Fixed by https://github.com/image-rs/image-tiff/pull/268
